packages/zowe-explorer/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 - **Breaking:** Removed the `zowe.uss.ZoweUSSNode.open` command in favor of using `vscode.open` with a USS URI. See the [FileSystemProvider wiki page](https://github.com/zowe/zowe-explorer-vscode/wiki/FileSystemProvider#file-paths-vs-uris) for more information on USS URIs. [#2207](https://github.com/zowe/zowe-explorer-vscode/issues/2207)
 - Added the `onResourceChanged` function to the `ZoweExplorerApiRegister` class to allow extenders to subscribe to any changes to Zowe resources (Data Sets, USS files/folders, Jobs, etc.). See the [FileSystemProvider wiki page](https://github.com/zowe/zowe-explorer-vscode/wiki/FileSystemProvider) for more information on Zowe resources.
 - Added the `addFileSystemEvent` function to the `ZoweExplorerApiRegister` class to allow extenders to register their FileSystemProvider "onDidChangeFile" events. See the [FileSystemProvider wiki page](https://github.com/zowe/zowe-explorer-vscode/wiki/FileSystemProvider) for more information on the FileSystemProvider.
+- Refactored the initialization, parsing and management of favorited nodes in Zowe Explorer. [#2026](https://github.com/zowe/zowe-explorer-vscode/issues/2026)
```

Comment:
In hindsight I was definitely being verbose here - sorry, bad habit 😋 

I rephrased it to "Refactored behavior and management of Favorites in Zowe Explorer" since nodes are a bit too developer-specific for the primary changelog. 

In this branch I've also adjusted the UX of session favorites to be consistent for each tree, so considering this and the state synchronization, I added "behavior" so users aren't surprised by some changes that weren't listed.